### PR TITLE
add migration guide

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] `npm test` passes
- [x] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/installation.handlebars [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
   <h1 class="large-title">Installation</h1>
 
-  <div class="align-center">
+  <h2><a href="./MigratingtoAdoptOpenJDKfromOracleJava.pdf" target="_blank">Migration Guide</a></h2>
+
+  <div class="align-center
+  ">
```
Comment (karianna) [inline src/handlebars/installation.handlebars]: typo new line

File: src/handlebars/support.handlebars [hunk 2]
```diff
@@ -8,12 +8,12 @@
   <div class="align-left support">
 
     <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#community-support" class="anchor">
-      <img src="dist/assets/anchor.png" id="anchor1" name="anchor" alt="anchor icon"></a>
+        <img src="dist/assets/anchor.png" id="anchor1" name="anchor" alt="anchor icon"></a>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: wrong whitespace?

File: src/handlebars/support.handlebars [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
       <h2 class="bold"><a id="support-levels">Community Support</a></h2>
     </div>
     <div class="margin-bottom">
       <p>
-        Read <a href="https://medium.com/@javachampions/java-is-still-free-2-0-0-6b9aa8d6d244">Java Is Still Free (2.0.0)</a>
+        Read <a href="https://medium.com/@javachampions/java-is-still-free-c02aef8c9e04">Java Is Still Free</a>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: You need to rebase I think

File: src/handlebars/support.handlebars [hunk 4]
```diff
@@ -27,10 +27,21 @@
         Of course, if the problem arises from the way we build and test the code we can fix that directly.
         For higher levels of assurance you should contact commercial companies offering support.
       </p>
+
+    </div>
+
+    <div class="anchor" onmouseover="document.getElementById('anchor2').style.visibility='visible'" onmouseout="document.getElementById('anchor2').style.visibility='hidden'"><a href="#migration-guide" class="anchor">
+        <img src="dist/assets/anchor.png" id="anchor2" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="support-levels">Migration Guide</a></h2>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: wrong id on `a` tag

File: src/handlebars/support.handlebars [hunk 5]
```diff
@@ -46,13 +57,13 @@
         <tr>
           <td><a target="_blank" href="https://www.jclarity.com/"><img src="dist/assets/jclarity.png" alt="jClarity Logo"></a></td>
           <td><strong>Hotspot VM</strong> - jClarity employs several of the founders of AdoptOpenJDK
-            and key engineers on the AdoptOpenJDK project. See <a href="https://www.jclarity.com/adoptopenjdk-support/" target="_blank">jClarity's AdoptOpenJDK Support</a> for more details.</td>
+            and key engineers on the AdoptOpenJDK project. Contact <a href="mailto:support@jclarity.com">support@jclarity.com</a> for more details.</td>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: again I think this is a rebase issue

File: src/handlebars/support.handlebars [hunk 6]
```diff
@@ -62,10 +73,12 @@
         <a href="https://www.eclipse.org/openj9/" target="_blank">Eclipse OpenJ9</a>,
         and <a href="https://sap.github.io/SapMachine/" target="_blank">SAP Machine</a>.
         </br>
-        </br>OpenJDK provides a new feature release every six months, and a maintenance/security update based upon each active release every three months. We will follow this schedule for publishing binary releases from AdoptOpenJDK to ensure you get the latest,
+        </br>OpenJDK provides a new feature release every six months, and a maintenance/security update based upon each active release every three months. We will follow this schedule for publishing binary releases from AdoptOpenJDK to ensure you
+        get the latest,
```
Comment (karianna) [inline src/handlebars/support.handlebars]: whitepace new line?

File: src/handlebars/support.handlebars [hunk 7]
```diff
@@ -62,10 +73,12 @@
...
         most secure builds.
         </br>
-        </br>In addition, every three years one feature release will be designated as the Long Term Supported (LTS) release. We will produce LTS releases for at least four years. This assurance will allow you to stay on a well-defined code stream, and give
+        </br>In addition, every three years one feature release will be designated as the Long Term Supported (LTS) release. We will produce LTS releases for at least four years. This assurance will allow you to stay on a well-defined code stream,
+        and give
```
Comment (karianna) [inline src/handlebars/support.handlebars]: whitespace new line?

**Comment by gdams** [comment]
updated PTAL

**Comment by kristi** [comment]
The Migration guide link appears at https://adoptopenjdk.net/installation.html but I get a 404 when I click on the link.

**Comment by karianna** [comment]
> The Migration guide link appears at https://adoptopenjdk.net/installation.html but I get a 404 when I click on the link.

We're looking at it

**Comment by ddoliver** [comment]
A link to IcedTea-Web doesn't work. I get a 404 when clicking on https://adoptopenjdk.net/adoptopenjdk/icedtea-web  from the migration guide.

**Comment by karianna** [comment]
> A link to IcedTea-Web doesn't work. I get a 404 when clicking on https://adoptopenjdk.net/adoptopenjdk/icedtea-web from the migration guide.

Fixed in https://github.com/AdoptOpenJDK/openjdk-website/pull/442

**Pull request merged**